builders/interoperability/xcm/send-execute-xcm.md

```diff
@@ -180,6 +180,75 @@ Now that you have the values for each of the parameters, you can write the scrip
 
 Once the transaction is processed, the 0.1 DEV tokens should be withdrawn from Alice's account along with the associated XCM fees, and the destination account should have received 0.1 DEV tokens in their account. A `polkadotXcm.Attempted` event will be emitted with the outcome.
 
+
+### Test an XCM Message with the XCM Dry Run API {: #test-an-xcm-message-with-the-dry-run-api }
+
+The XCM Dry Run API is an easy and convenient way to test the integrity of your XCM message without incurring any transaction fees. The XCM Dry Run API can be accessed from the [Runtime Calls](https://polkadot.js.org/apps/#/runtime){target=\_blank} tab of the **Developer** section of Polkadot.js Apps. 
+
+#### Dry Run Call API Method {: #dry-run-call-api-method }
```

Comment:
I believe this is not XCM-specific tho right? This section works for any call outside of XCM. So maybe moving this to another section is better?  Somewhere in the Substrate API for Moonbeam? Maybe even a new section on testing or something.

We should discuss

Reply:
Added a section to the Polkadot.js API page that's more general purpose for testing calls. It does seem like the dry Run method is primarily targeted at testing XCMs, although it works with arbitrary calls.